model-optimizer/extensions/front/ExpandDimsToUnsqueeze.py

```diff
@@ -35,5 +35,10 @@ def find_and_replace_pattern(self, graph: Graph):
                 expand_dims_node.in_port(0).get_connection().set_destination(unsqueeze_node.in_port(0))
                 expand_dims_node.out_port(0).get_connection().set_source(unsqueeze_node.out_port(0))
                 unsqueeze_node.in_port(1).connect(unsqueeze_dims_node.out_port(0))
+            elif len(expand_dims_node.in_nodes()) == 2:
```

Comment:
please just make a comment here that this case corresponds to ONNX Unsqueeze-13.